spec.html

```diff
@@ -4764,6 +4764,34 @@ <h1>GetFunctionRealm ( _obj_ )</h1>
         <p>Step 5 will only be reached if _obj_ is a non-standard function exotic object that does not have a [[Realm]] internal slot.</p>
       </emu-note>
     </emu-clause>
+
+    <emu-clause id="sec-copydataproperties" aoid="CopyDataProperties">
+      <h1>CopyDataProperties (_target_, _source_, _excluded_)</h1>
+      <p>When the abstract operation CopyDataProperties is called with arguments _target_, _source_ and _excluded_, the following steps are taken:</p>
+      <emu-alg>
+        1. Assert: Type(_target_) is Object.
+        1. Assert: _excluded_ is a List of property keys.
+        1. If _source_ is *undefined* or *null*, return _target_.
+        1. Else,
+          1. Let _from_ be ! ToObject(_source_).
+          1. Let _keys_ be ? _from_.[[OwnPropertyKeys]]().
+        1. For each element _nextKey_ of _keys_ in List order, do
+          1. Let _found_ be *false*.
+          1. For each element _e_ of _excluded_ in List order, do
+            1. If SameValue(_e_, _nextKey_) is true, then
```

Comment:
Hopefully this satisfies https://github.com/tc39/proposal-object-rest-spread/issues/44 @anba